doc/data/messages/n/nonlocal-and-global/good.py

```diff
@@ -1 +1,19 @@
-# This is a placeholder for correct code for this message.
+NUMBER = 42
+
+
+def create_number_updater():
+    previous = None
+
+    def update_number(number):
+        global NUMBER
+        NUMBER = number
+        print(f"New global number is: {NUMBER}")
+
+        nonlocal previous
+        print(f"Previous global number was: {previous}")
+        previous = number
+    return update_number
+
+
+update_number = create_number_updater()
+update_number(24)
```

Comment:
```suggestion
NUMBER = 42
NUMBER = 24
```

Reply:
@DanielNoord, do you mind giving your opinion on this ?

Reply:
Shouldn't the good example use either `global` or `nonlocal`? The warning is about using both and you needing to decide which it should not be, not remove both of them.